app/scripts/proactive/view/FileBrowserView.js

```diff
@@ -113,7 +123,7 @@ define(
             }
             $.ajax({
                 url: that.dataspaceRestUrl + encodeURIComponent(pathname),
-                data: { "comp": "list" },
+                data: { "comp": "list", "includes": '*' + this.filterValue + '*'},
```

Comment:
I would rather have the filterValue directly applied as includes. The default value of filterValue can be '*'